src/replica/duplication/mutation_batch.h

```diff
@@ -45,7 +45,7 @@ class mutation_buffer : public prepare_list
     void commit(decree d, commit_type ct) override;
 
 private:
-    perf_counter_wrapper _counter_dulication_mutation_loss_count;
+    METRIC_VAR_DECLARE_gauge_int64(dup_recent_lost_mutations);
```

Comment:
Is the "recent" word necesary?

Reply:
Yes, actually this metric is `set` with the number of lost mutations, thus it's `recent`.

Reply:
To record all change values, will would be great to use counter? we can improve it in the future.

Reply:
OK.